### DISCUSSION TITLE
youtube

### DISCUSSION THREAD

**Discussion by pokernet** [AdguardTeam/AdGuardHome#2909] [Q&A]
i get this on youtube 

Video unavailable
This video is restricted. Please check the Google Workspace administrator and/or the network administrator restrictions.

even if i cancel all restriction for pc or all global restriction 

its bloack youtube for 50% of the clips

**Comment by Aikatsui** [comment]
Why you think that is adguardhome issue? Did you turn on `enforce safe search` recently?

**Comment by pokernet** [comment]
,hi

Yes i turn it on and then I turn it off

. If I change the dns of the pc to 8.8.8.8 all work ok
If I use gh dns then youtube start blocking .

I try set all to.off o. The gh . And it's not helping

**Comment by ameshkov** [comment]
You either use "Enforce Safe Search" or a DNS server that enforces restricted mode on YouTube.
